.circleci/config.yml

```diff
@@ -5,6 +5,20 @@ executors:
       - image: circleci/openjdk:17-buster-node-browsers-legacy
 
 jobs:
+  gradle_build:
+    executor: docker-executor
+    environment:
+      JVM_OPTS: -Xmx512m
+      TERM: dumb
+
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.14
+      - run:
+          name: Checkstyle and spotless
+          command: ./gradlew build
```

Comment:
can we add this as a step in existing jobs?

Reply:
We can add it in existing job but then pass fail won't be shown cause docker image will be built with bootjar so.